src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Should we add:
```xml
<UseSharedCompilation>false</UseSharedCompilation>
<GenerateDocumentationFile>False</GenerateDocumentationFile>
<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
<WarningLevel>4</WarningLevel>
<SuppressNETCoreSdkPreviewMessage>True</SuppressNETCoreSdkPreviewMessage>
<SignAssembly>false</SignAssembly>
```

Reply:
I added the reference to `common.props` file